### PR TITLE
Modify logging module

### PR DESCRIPTION
**Pull request by LuQQiu**

### CODE REVIEW

File: alluxiofs/core.py [hunk 1]
```diff
@@ -120,6 +116,17 @@ def __init__(
             if preload_path is not None:
                 self.alluxio.load(preload_path)
 
+        if "log_level" in test_options:
```
Comment (lucyge2022) [inline alluxiofs/core.py:119]: think u should include in the options

File: alluxiofs/core.py [hunk 2]
```diff
@@ -120,6 +116,17 @@ def __init__(
...
+            log_level = test_options["log_level"].upper()
+            if log_level == "DEBUG":
+                logger.setLevel(logging.DEBUG)
+            elif log_level == "INFO":
+                logger.setLevel(logging.INFO)
+            elif log_level == "WARN" or log_level == "WARNING":
+                logger.setLevel(logging.WARN)
+            else:
+                logger.warning(f"Unsupported log level: {log_level}")
```
Comment (lucyge2022) [inline alluxiofs/core.py:128]: and lets set a default warning level to be ERROR (logging has ERROR level correct? )

File: alluxiofs/core.py [hunk 3]
```diff
@@ -164,16 +171,26 @@ def fallback_wrapper(self, path, *args, **kwargs):
             path = self._strip_alluxio_protocol(path)
             try:
                 if self.alluxio:
-                    self.logger.debug(f"calling {alluxio_impl.__name__}")
-                    return alluxio_impl(self, path, *args, **kwargs)
+                    start_time = time.time()
+                    res = alluxio_impl(self, path, *args, **kwargs)
+                    self.logger.debug(
+                        f"Successfully called {alluxio_impl.__name__} against alluxio server with args ({args}), kwargs ({kwargs}) for {time.time() - start_time} ms"
```
Comment (lucyge2022) [inline alluxiofs/core.py:177]: this time.time() is in sec

**Review by lucyge2022 — APPROVED** [review]

File: alluxiofs/core.py (hunk 1 above)
Comment (LuQQiu) [reply alluxiofs/core.py:119]: alluxiofs is an inner Python module
For each Python process, the end launch program should define to log level, and all the inner Python module inherit the logging configuration.
E.g. in our benchmark entry point, we should define the log level and log file there

File: alluxiofs/core.py (hunk 2 above)
Comment (LuQQiu) [reply alluxiofs/core.py:128]: logging usually don't have error, if it's error, the program should directly error out and failed
but they do have error log level

File: alluxiofs/core.py (hunk 1 above)
Comment (lucyge2022) [reply alluxiofs/core.py:119]: no I meant to say why not use the dict "options" ? its also user defined at time of instantiating alluxiofs instance right?

**Comment by LuQQiu** [comment]
@ChunxuTang @lucyge2022 PTAL thanks!
Use bench.py test out the logging
By default the logging should not set any config so that python parent modules' logging config can be passed in.
Add a test_option {"log_level": "debug"} to explicitly override alluxio related logging to debug

**Review by lucyge2022 — APPROVED** [review]
LGTM

**Pull request merged**